ouroboros-consensus/docs/ChainSync.md

```diff
@@ -8,9 +8,10 @@ the most _recent_ intersection (could be genesis).
 0.  We will never roll back more than `k` blocks.
 1.  This means we are not interested in upstream nodes whose chains fork from
     our chain more than `k` blocks ago.
-    (If an upstream node does not satisfy this condition, we will disconnect;
-    it will be the responsibility of the network layer to decide whether or not
-    to try reconnecting to such nodes later.)
+    - If an upstream node does not satisfy this condition, we will disconnect
```

Comment:
End this sentence with a period, for consistency with the other two points :grin: